### PR TITLE
update: remove boiler plate category and insert new tools on web dev category

### PR DESCRIPTION
**Pull request by simaraconceicao**
## Adding new section named boiler plate with tools to generate projects

- [ x ] Link to library is included 
- [ x ] Name of library is included
- [ x] Name and link are spelled correctly

Happy hacktoberfest! 👩🏾‍💻😍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,3 +201,15 @@ Semantic UI is an open source UI framework.
...
+## Boiler Plate
+
+- [Vite](https://vitejs.dev/)
+Vite is a boiler plate to generate front-end projects with support to many javascript frameworks.
+
+- [Serverless](https://www.serverless.com/)
+Serverless is a framework to generate backend projects with AWS Lambda.
+
+- [Create React App](https://create-react-app.dev/)
+Create React App is a boiler plate to generate React Apps.
```
Comment (jdwilkin4) [inline README.md:214]: I think these are good additions but let's place it under web dev tools and remove the Boiler Plate category. 👍

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by simaraconceicao** [comment]
Hey, thanks for the feedback. I did the changes. I hope you aprove.

**Review by jdwilkin4 — APPROVED** [review]
Hi @simaraconceicao ! 

Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**